server/utils/features.js

```diff
@@ -17,7 +17,6 @@ const features = {
     ),
     allOf(
       isEnv('preprod'),
-      enabledForCourts(PLYMOUTH_MAGS_COURT_CODE, 'B05PK'),
```

Comment:
is it no longer a requirement to have it enabled for plymouth?

Reply:
preprod - so removed court requirement to simplify granting access to the team